### DISCUSSION TITLE
Releasing a new version: 0.1.6

### DISCUSSION THREAD

**Discussion by kaplanelad** [loco-rs/loco#126] [Announcements]
🎉 We are thrilled to announce the release of Loco version 0.1.6! Explore the [changelog](https://github.com/loco-rs/loco/blob/master/CHANGELOG.md#v016) to discover exciting new features.

Highlights of this release include:

- Refactored configuration loader, allowing the use of defaults from environment variables. Developers can now seamlessly work with `<env>.local.yaml` during development.
- Introducing new hooks:`Hooks::before_run` and`Hooks::after_routes`. 
- Cors middleware
- Adding docker deployment generator by running the command: `cargo loco generate deployment`

**Comment by Anakin100100** [comment]
Awesome to see you working on a Rails like framework in Rust
